build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
```

Comment:
This file is actually itself kotlin (in script flavor, hence .kts instead of .kt)